app/javascript/components/get-version.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import { LoadingMessage, ErrorMessage } from '@messages/default-messages';
+
+var lastValue = null;
+
+const getVersion = gql`
+  {
+    schemaSha
+  }
+`;
+
+export const Version = () => (
+  <Query query={getVersion}>
+    {({ loading, error, data }) => {
+      if (loading) {
+        return <LoadingMessage />;
```

Comment:
I think we should probably return `null` for both loading and error. No reason for this component to create dom elements.